server/chat-jsx.tsx

```diff
@@ -28,6 +28,7 @@ export interface PSElements extends preact.JSX.IntrinsicElements {
 	spotify: {src: string};
 	username: {name?: string, class?: string};
 	psicon: {pokemon: string} | {item: string} | {type: string} | {category: string};
+	center: Record<string, unknown>;
```

Comment:
ಠ_ಠ

Reply:
```suggestion
	center: {};
```

Reply:
the face doesn't really help with explaining why it's wrong, but i am doing this

Reply:
Also, `{}` isn't right either; I get this error:

`31:10  error  Don't use `{}` as a type. `{}` actually means "any non-nullish value".`

Reply:
Unfortunately, in this case, it is in fact a correct value. Probably changing it to `{class?: string}` will shut up the linter, though.

Reply:
The face is because `Record<string, unknown>` is nothing like the other entries, like `{src: string}`, which are lists of attributes. If you are going to do something like this (which seems intended to remove all validation), you should at least ask me if it's a good idea.

In general, I would prefer that if you are going to shut up a validator of some sort in the PS codebase, you should first ask me if there is a better alternative.

Reply:
Is that not what the point of a pull request is?

Reply:
Regardless, fixing

Reply:
A pull request is an opportunity for people to catch your mistakes, but you shouldn't rely on the idea that I will carefully read every single line, especially of longer pull requests. If you intentionally do something dangerous, you should comment on that part of the PR to ask for special attention (such as asking me if there are better ways).

I do that on my pull requests, you know; I'll often point out specific lines where I'm doing weird things.